packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -14,17 +12,21 @@ import * as PackageManager from '../../PackageManager';
 import { validateGitStatusAsync } from '../utils/ProjectUtils';
 
 import log from '../../log';
-import prompt from '../../prompt';
+import prompt, { Question } from '../../prompt';
 import { loginOrRegisterIfLoggedOut } from '../../accounts';
 
 const EXPO_APP_ENTRY = 'node_modules/expo/AppEntry.js';
 
-async function warnIfDependenciesRequireAdditionalSetupAsync(projectRoot: string) {
+async function warnIfDependenciesRequireAdditionalSetupAsync(projectRoot: string): Promise<void> {
   const { exp, pkg: pkgJson } = await ProjectUtils.readConfigJsonAsync(projectRoot);
+  if (!exp || !pkgJson) {
```

Comment:
Instead of `ProjectUtils.readConfigJsonAsync`, please use `ConfigUtils.readConfigJsonAsync` (from `@expo/config`), which never returns `null`.